source/ways-of-working/troubleshooting/index.html.md.erb

```diff
@@ -210,3 +254,17 @@ And then reference this branch within your repo's [Jenkinsfile](https://github.c
    # search for specific image
    kubectl logs -n flux-system -l app=kustomize-controller | grep <namespace>
    ```
+
+## Connecting to AKS Clusters
```

Comment:
aww this has moved down a lot, I use it all the time, I wonder if it should have it's own page

Reply:
If you have a link, it should still work on same link :)